model/Concept.php

```diff
@@ -508,7 +508,13 @@ public function getProperties()
                 $sprop = "<$prop>";
             }
 
-            if (!in_array($prop, $this->DELETED_PROPERTIES) || ($this->isGroup() === false && $prop === 'skos:member')) {
+            // delete notations unless configured to show them
+            $deleted = $this->DELETED_PROPERTIES;
+            if ($sprop === 'skos:notation' && !$this->vocab->getConfig()->getShowNotationAsProperty()) {
```

Comment:
No reason to check `$sprop === 'skos:notation'` here - and if the whole block is moved, as suggested above, it cannot be done anyway as `$prop` is only set within the foreach loop.

Reply:
Fixed with this: https://github.com/NatLibFi/Skosmos/pull/1198#pullrequestreview-750353342

Reply:
Done